x/tokenfactory/keeper/msg_server.go

```diff
@@ -0,0 +1,233 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/titantkx/titan/x/tokenfactory/types"
+)
+
```

Comment:
Missing support `UpdateParams` that allow config tokenfactory params by gov module.

REF: like distribution module in sdk : https://github.com/tokenize-titan/cosmos-sdk/blob/bbf75c8970e38bd7cfee84ad064ebbf18775fdfc/x/distribution/keeper/msg_server.go#L118

Reply:
Added